tests/test_conditions.py

```diff
@@ -77,6 +83,32 @@ def test_weights():
     assert dist.components[0].loc == pytest.approx(2, rel=0.1)
 
 
+@dataclass
+class Uniform:
```

Comment:
Would move to distributions folder, make instance of Distribution class. You can make an issue for that if you don't want to do it now.

Reply:
Made this issue: https://github.com/oughtinc/ergo/issues/312

Was really just going for the minimum needed for testing here, we might want to add more to it for it to be a real distribution